### PR TITLE
Group edit

### PR DESCRIPTION
**Pull request by OliverMclachlan**
Now a list of members now appear when you click on a group

### CODE REVIEW

File: src/Facade/ResourceBuilders/GroupResourceBuilder.cs [hunk 1]
```diff
@@ -3,15 +3,34 @@
...
+            var members = model.Members;
+            var membersResources = new List<MemberResource>();
+
+            foreach (Member member in members)
+            {
+                membersResources.Add((MemberResource)this.memberResourceBuilder.Build(member , claims));
+            }
+
+
+            return new GroupResource {
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/GroupResourceBuilder.cs:27]: some blank lines

File: src/Facade/ResourceBuilders/GroupResourceBuilder.cs [hunk 2]
```diff
@@ -3,15 +3,34 @@
...
+            var membersResources = new List<MemberResource>();
+
+            foreach (Member member in members)
+            {
+                membersResources.Add((MemberResource)this.memberResourceBuilder.Build(member , claims));
+            }
+
+
+            return new GroupResource { 
+                                             Active = model.Active,
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/GroupResourceBuilder.cs:28]: indentation? or is github just rendering this funny?

File: src/Facade/ResourceBuilders/MemberResourceBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    using Linn.Authorisation.Domain.Groups;
+    using Linn.Authorisation.Resources;
+    using Linn.Common.Facade;
+
+    using Org.BouncyCastle.Crypto.Modes.Gcm;
+
+    public class MemberResourceBuilder : IBuilder<Member>
+    {
+
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/MemberResourceBuilder.cs:14]: blan line

File: src/Facade/ResourceBuilders/MemberResourceBuilder.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        public object Build(Member model, IEnumerable<string> claims)
+        {
+            if (model is IndividualMember)
+            {
+                return new MemberResource { MemberUri = ((IndividualMember)model).MemberUri };
+            }
+            //return new MemberResource {Group = ((GroupMember)model).Group }
+
+            return new MemberResource{};
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/MemberResourceBuilder.cs:23]: this looks odd - why just return an empty MemberResource? the commented out line on 21 looks more reasonable

File: src/Resources/GroupResource.cs [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
     public class GroupResource
     {
         public string Name { get; set; }
 
         public bool Active { get; set; }
 
         public int Id { get; set; }
+
+        public IEnumerable<MemberResource> Members { get; set; }
+
```
Comment (lewisrenfrew) [inline src/Resources/GroupResource.cs:15]: blank line

File: src/Service.Host/client/src/components/Group.js [hunk 6]
```diff
@@ -51,6 +55,21 @@ function Group() {
...
     };
 
+    const getMembers = member => {
+        if (!employees?.items.find(i => member.memberUri === i.href)) {
+            return;
+        }
+
+        const employee = employees?.items.find(i => member.memberUri === i.href);
+
+        // eslint-disable-next-line consistent-return
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/Group.js:65]: Do we need this eslint-disable? i would just remove it and let pretter format the file and see what happens?

File: src/Service.Host/client/src/components/Group.js [hunk 7]
```diff
@@ -96,6 +115,13 @@ function Group() {
                     autoHideDuration={5000}
                     message="Save Successful"
                 />
+
+                <Grid item xs={12}>
+                    <Typography variant="h5">Group Members</Typography>
+
+                    <List>{group?.members?.map(getMembers)}</List>
+                    {/* <List>{memberNames?.map(displayMembers)}</List> */}
```
Comment (lewisrenfrew) [inline src/Service.Host/client/src/components/Group.js:123]: delete?

File: tests/Integration/Integration.Tests/GroupModuleTests/WhenPutting.cs [hunk 8]
```diff
@@ -21,7 +21,7 @@ public class WhenPutting : ContextBase
         [SetUp]
         public void SetUp()
         {
-            this.updatedResource = new GroupResource { Name = "new.Group.Test.Name", Active = false, Id = 30} ;
+            this.updatedResource = new GroupResource { Name = "new.Group.Test.Name", Active = false, Id = 30};
```
Comment (lewisrenfrew) [inline tests/Integration/Integration.Tests/GroupModuleTests/WhenPutting.cs:24]: should be a space in between the number '30' and the closing curly bracket

File: src/Facade/ResourceBuilders/MemberResourceBuilder.cs [hunk 9]
```diff
@@ -0,0 +1,33 @@
+﻿namespace Linn.Authorisation.Facade.ResourceBuilders
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Linn.Authorisation.Domain.Groups;
+    using Linn.Authorisation.Resources;
+    using Linn.Common.Facade;
+
+    using Org.BouncyCastle.Crypto.Modes.Gcm;
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/MemberResourceBuilder.cs:10]: this probably sneaked in unintentionally?

**Review by lewisrenfrew — APPROVED** [review]

File: src/Service.Host/client/src/components/Group.js (hunk 6 above)
Comment (OliverMclachlan) [reply src/Service.Host/client/src/components/Group.js:65]: i tried deleting it and it didn't build it's got something to do with the empty return for old employees

**Pull request merged**